src/runtime-rs/Cargo.toml

```diff
@@ -4,3 +4,6 @@ members = [
     "crates/shim-ctl",
 ]
 
+[patch.crates-io]
+fuse-backend-rs = { path = "/root/repo/fuse-backend-rs" }
```

Comment:
This line will be removed after merging [cloud-hypervisor/fuse-backend-rs#159](https://github.com/cloud-hypervisor/fuse-backend-rs/pull/159).